index.html

```diff
@@ -1,238 +1,276 @@
-        <li>Back End Dev</li>
-        <li><img src="Images/Counter.png" alt="Counter"></li>
-        <li>2015</li>
-    </ol>
-    
-    <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
-    
-    <ul>
-        <li>html</li>
-        <li>css</li>
-        <li>javaScipt</li>
-    </ul>
-    <button type="button" class="button"> See Project</button>
-
-</div>
-
-
-<div class="project">
-    <img src="Images/SnapshootPortfolio3.png" alt="SnapshootPortfolio3">
-    <h2>Multi-Post <br>Stories</h2>
-    
- <ol>
-        <li>CANOPY</li>
-        <li><img src="Images/Counter.png" alt="counter"></li>
-        <li>Back End Dev</li>
-        <li><img src="Images/Counter.png" alt="counter"></li>
-        <li>2015</li>
- </ol>
-
- <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
-
- <ul>
-    <li>html</li>
-    <li>css</li>
-    <li>javaScipt</li>
-</ul>
-
-<button type="button" class="button"> See Project</button>
-
-</div>
-</section>
-
-
-
-<section class="about">
-    <h2>About <br> Myself</h2>
-
-    <p>Hello I'm a software developer! I can help <br> you build a product , feature or website <br> Look through some of my work and <br> experience!If you like what you see and <br> have a project you need coded, don't <br> hestiate to contact me. </p>
-
-   <h3>LET'S CONNECT</h3>
-
-<ul class="icons">
-    <li>
-        <a href="#"><img src="./Images/Vector.png" alt="Vector"></a>
-    </li>
-    <li>
-        <a href="#"><img src="./Images/Linkedinicon.png" alt="Linkedin icon"></a>
-    </li>
-
-    <li>
-        <a href="#"><img src="./Images/Vector1.png" alt="Vector 1"></a>
-    </li>
-    <li>
-        <a href="#"><img src="./Images/Vector2.png" alt="Vector 2"></a>
-    </li>
-    <li>
-        <a href="#"><img src="./Images/Vector3.png" alt="Vector 3"></a>
-    </li>
-    
-</ul>
-
-<button type="button" class="button"> Get my resume</button>
-
-</section>
+    <header class="header">
+        <a href="#">My Logo</a>
+        <nav>
+            <img src="Images/Union.png" alt="hamburger">
+            <ul class="desktop-links">
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="body" id="portfolio">
+        <h1>I'm Gleen <br> Glad to see <br class="br"> you!</h1>
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2>LET'S CONNECT</h2>
+
+        <ul>
+            <li>
+                <a href="#"><img src="./Images/Vector.png" alt="Vector"></a>
+            </li>
+            <li>
+                <a href="#"><img src="./Images/Linkedinicon.png" alt="Linkedin icon"></a>
```

Comment:
- [ Optional] Kindly use the appropriate background image both for desktop and mobile versions. I think, you are using the bg image for mobile version in the desktop version, there is no background image in the mobile version. 
#### Design
![headline-bg-img-figma](https://user-images.githubusercontent.com/44057863/167268332-768c4cef-6d1e-4dbd-ab3a-a3de43ab0fe6.png)
#### Imp
![headline-bg-img-impl](https://user-images.githubusercontent.com/44057863/167268337-31f7bbaa-201a-4e8b-bf99-439e2cc2a784.png)
l